api/integrations/github/migrations/0004_auto_20240527_1503.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.25 on 2024-05-27 15:03
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
You should be able to combine this migration with the other one.

Reply:
I usually try to avoid extra migrations, I prefer to keep it separated in this case.